root/search/components/UrlResults.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import type {ResultsPropsT, SearchResultT} from '../types';
+import PaginatedResults from '../../components/PaginatedResults';
+import Table from '../../components/Table';
+import {defineLinkColumn} from '../../utility/tableColumns';
+
+import ResultsLayout from './ResultsLayout';
+
+const UrlResults = ({
+  form,
+  lastUpdated,
+  pager,
+  query,
+  results,
+}: ResultsPropsT<UrlT>):
+React.Element<typeof ResultsLayout> => {
+  const columns = React.useMemo(
+    () => {
+      const nameColumn = defineLinkColumn<SearchResultT<UrlT>>({
```

Comment:
Should we be adding `target="_blank" rel="noopener noreferrer"` to these?

Reply:
We don't currently do it on the report list at root/report/components/UrlList.js but that doesn't mean we shouldn't. I added the change (to both files) in a second commit. That said, https://chromestatus.com/feature/6140064063029248 suggests in most browsers `noopener` is not required if you already have `_blank`.